sql_extensions/migrations/special-queries.sql

```diff
@@ -47,15 +47,54 @@ WHERE
 ORDER BY
     (data -> 'market_metadata' ->> 'market_id')::NUMERIC,
     (data ->> 'transaction_version')::NUMERIC DESC,
-    (data ->> 'event_index')::NUMERIC DESC;
+    (data -> 'state_metadata' ->> 'market_nonce')::NUMERIC DESC;
```

Comment:
Same as below, `market_nonce` will always be in order, and I'm not sure if `event_index` is processed in order